README.md

```diff
@@ -234,16 +234,16 @@ It's possible that your topic and system are entirely ok with losing some messag
     - `KAFKA_CLIENT_CERT_KEY`
     - `KAFKA_TRUSTED_CERT`
 
-2. Start `Kaffe.Producer` with your application
-
-      ```elixir
-      worker(Kaffe.Producer, [])
-      ```
-
 ### Producing to Kafka
 
 Currently only synchronous message production is supported.
 
+You'll need to start the `Kaffe.Producer`, this may be best suited within your applications callback:
+
+```elixir
+Kaffe.Producer.start_producer_client()
```

Comment:
@rawkode This should be started if there is a [producer config](https://github.com/spreedly/kaffe/blob/master/lib/kaffe.ex#L18). Is that not working for you?